templates/terraform/main.tf

```diff
@@ -0,0 +1,81 @@
+provider "aws" {
+  region              = "${var.region}"
+}
+
+# {{ if .Config.Infrastructure.AWS.Terraform.RemoteState }}
+# Store remote state in S3
+resource "aws_s3_bucket" "terraform_remote_state" {
```

Comment:
Can you remove this stuff please? It's already created by the bootstrap.

Reply:
kk removed.